src/index.html

```diff
@@ -6,11 +6,66 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
+
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form-wrapper"
+      data-qa="big"
+    >
+      <label
+        for="searchOne"
+        class="label-image"
+      >
+        <img
+          alt="search icon"
+          src="images/Search.svg"
+          class="search-image"
+        >
+      </label>
+      <input
+        type="text"
+        name="search-large"
+        id="searchOne"
+        class="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form-wrapper form-wrapper--small"
+      data-qa="small"
+    >
+      <label
+        for="searchTwo"
+        class="label-image"
+      >
+        <img
+          alt="search icon"
+          src="images/Search.svg"
+          class="search-image"
+        >
+      </label>
+
+      <input
+        type="text"
+        name="search-small"
+        id="search-wrapper--small"
```

Comment:
The 'id' attribute should have a value of 'searchTwo' to match the 'for' attribute in the corresponding label.